mne/time_frequency/tfr.py

```diff
@@ -271,7 +273,8 @@ def _compute_tfr(epoch_data, frequencies, sfreq=1.0, method='morlet',
     zero_mean : bool | None, defaults to None
         None means True for method='multitaper' and False for method='morlet'.
         If True, make sure the wavelets have a mean of zero.
-    time_bandwidth : float, defaults to 4.0 (3 tapers)
+    time_bandwidth : float, defaults to None
+        If method=multitaper, will be set to 4.0 (3 tapers).
```

Comment:
"If none and method=" would be clearer. Also this should be below the description of the argument (last line), but is the first line now.